airbyte-ci/connectors/base_images/base_images/java/bases.py

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from __future__ import annotations
+
+from typing import Callable, Final
+
+import dagger
+from base_images import bases, published_image
+from base_images import sanity_checks as base_sanity_checks
+from base_images.python import sanity_checks as python_sanity_checks
+from base_images.root_images import AMAZON_CORRETTO_21_AL_2023
+from base_images.utils.dagger import sh_dash_c
+
+
+class AirbyteJavaConnectorBaseImage(bases.AirbyteConnectorBaseImage):
+    # TODO: remove this once we want to build the base image with the airbyte user.
```

Comment:
This I expect mimics the current setup?

Reply:
Yes the current user is root. The parent class sets it to airbyte, so I override the `USER` with `root` to keep v1 expected user be `root`.